static/css/style.css

```diff
@@ -828,3 +867,16 @@ img.showcase-image-edit {
     font-size: 1.1rem;
     }
 }
+
+@media (max-width:650px){
+    .team-menu-dropdown-container {
+        position: relative;
+        right: 0;
+    }
+    .teams-dropdown {
+        transform: translate3d(0px, 38px, 0) !important;
```

Comment:
It's overwriting a Bootstrap class that uses `!important`. So I am not sure if specificity will solve this here?!

Reply:
Leave it, if it's overwriting a BS class then... Otherwise use the Inspect tool on DevTools to see how they're targeting the path for it. Just leave it in this instance.